tools/docker/README.md

```diff
@@ -1,16 +1,15 @@
 This docker images is used for testing Chrome, Firefox, WebKitGTK and running
 other tasks on Taskcluster. When any of the files in this directory change, the
-images must be updated as well. Doing this requires you be part of the
-'webplatformtests' organization on Docker Hub; ping @foolip or @jpchase
-if you are not a member.
+images must be updated as well. Doing this requires you to have write
+permissions to the repository.
 
 The tag for a new docker image is of the form
-`webplatformtests/wpt:{current-version + 0.01}`
+`{current-version + 0.01}`
```

Comment:
This is now wrong again. Also I'd take this chance to just move to integer version numbers.